src/SampleMauiApplication/Platforms/Android/AndroidManifest.xml

```diff
@@ -3,4 +3,19 @@
 	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="31" />
 	<application android:allowBackup="true" android:icon="@mipmap/appicon" android:roundIcon="@mipmap/appicon_round" android:supportsRtl="true"></application>
 	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+	<queries>
+		<!-- For opening external websites required for social logins -->
+		<intent>
+			<action android:name="android.support.customtabs.action.CustomTabsService" />
+		</intent>
+		<!-- MediaPicker -->
+		<intent>
+			<action android:name="android.media.action.IMAGE_CAPTURE" />
+		</intent>
+		<!-- Maps -->
+		<intent>
+			<action android:name="android.intent.action.VIEW" />
```

Comment:
Is this needed for social login?  (Similar question to the Image Capture intent.

Reply:
Ah nope, just the CustomTabsService needed, my bad.

Reply:
Now worries, just want to try and keep this to "MVP" status, especially for permission/intent asks